docs/src/core/initial-setup.md

```diff
@@ -1,3 +1,57 @@
 # Initial Setup
 
+Laniakea requires a postgresql database, a D compiler, and the meson
+build tool to get up and running. If you don't already have those
+packages installed, install them now;
+
+$ sudo apt install postgresql-11 postgresql-11-debversion meson ldc libarchive libcurl cmake
+
+Create a build directory below the top-level directory that holds the
+meson.build file and call meson from it;
+
+$ mkdir build && cd build
+$ meson -Dwebgui=true -Drubicon=false -Dlighthouse=false \
```

Comment:
Since this document is generic-ish, it may not be good to disable a lot of modules, for example Lighthouse is pretty essential.
So for this example, it may be useful to build with all default-enabled modules.

Reply:
> The build still asks for python3-flask-cache. Perhaps you have to update the module internally?

Oh, didn't you change that back to Flask-Caching? If it's changed in requirements.txt, then it should have been picked up automatically by the Meson build script.